samples/CppArExample/CppArExample.pro

```diff
@@ -0,0 +1,75 @@
+#-------------------------------------------------
+#  Copyright 2019 ESRI
+#
+#  All rights reserved under the copyright laws of the United States
+#  and applicable international laws, treaties, and conventions.
+#
+#  You may freely redistribute and use this sample code, with or
+#  without modification, provided you include the original copyright
+#  notice and use restrictions.
+#
+#  See the Sample code usage restrictions document for further information.
+#-------------------------------------------------
+
+TEMPLATE = app
+
+QT += core gui opengl network positioning sensors qml quick multimedia
+CONFIG += c++11
+
+TARGET = CppArExample
+
+equals(QT_MAJOR_VERSION, 5) {
+    lessThan(QT_MINOR_VERSION, 12) { 
+        error("$$TARGET requires Qt 5.12.0")
+    }
+	equals(QT_MINOR_VERSION, 12) : lessThan(QT_PATCH_VERSION, 0) {
+		error("$$TARGET requires Qt 5.12.0")
+	}
+}
+
+ARCGIS_RUNTIME_VERSION = 100.6
+include($$PWD/arcgisruntime.pri)
+
+HEADERS += \
+    AppInfo.h \
+    CppArExample.h
+
+SOURCES += \
+    main.cpp \
+    CppArExample.cpp
+
+RESOURCES += \
+    qml/qml.qrc \
+    Resources/Resources.qrc
+
+OTHER_FILES += \
+    qml/*.qml \
+    qml/*.js
+
+#-------------------------------------------------------------------------------
+# AR configuration
+
+# The path to the ArcGIS runtime toolkit for Qt sources, corresponding to the files downloaded
+# from the GitHub repo: https://github.com/Esri/arcgis-runtime-toolkit-qt
+
+AR_TOOLKIT_SOURCE_PATH = # must be set to the path to toolkit sources
```

Comment:
you could consider adding a qmake message for if this is empty so it is obvious to a user that opens the example that they need to set this. Otherwise, it is a bit buried and might be easy to miss